grammars/css.cson

```diff
@@ -561,63 +561,113 @@
         'captures':
           '1':
             'name': 'punctuation.definition.entity.css'
+        'match': '''(?x)
+          (:)
+          (
+            link|target|visited                                     # location
+          | active|focus|hover                                      # user action
+          | default|disabled|enabled|read-only|read-write           # input control state
+          | checked|indeterminate                                   # input value state
```

Comment:
As I said in the comment of ee52174, this name "ui-state" itself doesn't make sense, that's why I want to remove it. And I think it should be safe even for themes that are relying on this class coz the related properties will still get the "pseudo-class" class.